src/libs/actions/IOU.ts

```diff
@@ -4829,7 +4845,7 @@ function sendInvoice(
         accountID: currentUserAccountID,
         amount: transaction?.amount ?? 0,
         currency: transaction?.currency ?? '',
-        comment: transaction?.comment?.comment?.trim() ?? '',
+        comment: parsedComment,
```

Comment:
We should not need the comment parameter as only the description will be used. We should clean this up in a follow up @dominictb @akinwale